.circleci/config.yml

```diff
@@ -47,6 +77,87 @@ jobs:
       - run:
           name: Run Checks
           command: bin/ci-checks-js.sh
+  android-device-checks:
+    docker:
+    - image: circleci/android:api-27-node8-alpha
+    steps:
+      - checkout
+      - run:
+          name: Checkout Gutenberg
+          command: |
+            git submodule init
+            git submodule update --recursive
+      - yarn-install
+      - run:
+          name: Accept Licenses
+          command: |
+            yes | sdkmanager --licenses > /dev/null
+            sleep 10
+            sdkmanager --update
+          background: true
+      - run:
+          name: Set Environment Variables
+          command: |
+            echo 'export TEST_RN_PLATFORM=android' >> $BASH_ENV
+            echo 'export TEST_ENV=sauce' >> $BASH_ENV
+      - build-android
+      - run:
+          name: Upload apk to sauce labs
+          command: |
+            curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" -X POST -H "Content-Type: application/octet-stream" https://saucelabs.com/rest/v1/storage/automattic/Gutenberg.apk?overwrite=true --data-binary @./android/app/build/outputs/apk/debug/app-debug.apk
+      - run:
+          name: Run Device Tests
+          command: yarn device-tests
+  ios-device-checks:
+    macos:
+      xcode: "10.0"
+    steps:
+    - checkout
+    - run:
+        name: Checkout Gutenberg
+        command: |
+          git submodule init
+          git submodule update --recursive
+    - run: yarn clean:install
```

Comment:
Why can't we use the `yarn-install` command here with caching? `yarn clean:install` is quite slow.

Reply:
I included this when I was trying to get rid of the architecture error I had mentioned, 🤔not sure if it's necessary now, I'll remove and see